File: Algorithms/Warmup/Plus Minus.java
/*
* Contributer : github.com/shikhar-07
* Email : shikharanandg@gmail.com
*/
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {

    public static void main(String[] args) {
    double negative=0;
  double positive=0;
  double zero=0;  
  Scanner k=new Scanner(System.in);
  int n=k.nextInt();
  int [] arr=new int[n];
  for(int m:arr)
  {
       m=k.nextInt();
       if(m==0)
       {
             zero++;
        }
       else if(m<0)
      {
             negative++;
       }
      else if(m>0)
     {
           positive++;
      }
  }
  System.out.println(positive/n);
  System.out.println(negative/n);
  System.out.println(zero/n);
    }
}

File: Algorithms/Warmup/Plus Minus.cpp
/*
   Contributer : github.com/jaswal72
   Email : shubhamjaswal772@gmail.com
*/
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main(){
    int n;
    float a=0,s=0,d=0;
    cin >> n;
    vector<int> arr(n);
    for(int arr_i = 0;arr_i < n;arr_i++){
       cin >> arr[arr_i];
        if(arr[arr_i]>0)
            ++a;
        if(arr[arr_i]<0)
            ++s;
        if(arr[arr_i]==0)
            ++d;
    }
    cout<<a/n<<"\n"<<s/n<<"\n"<<d/n<<"\n";
    return 0;
}
